cmd/plugins/memory-policy/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG GO_VERSION=1.23
```

Comment:
Since [this](https://github.com/containers/nri-plugins/pull/528) change, we better bump the go version here too ?

Reply:
Good catch! Thanks @fmuyassarov!